app/Lib/lang.php

```diff
@@ -1078,6 +1078,7 @@
   'er.input.blank' => 'Value cannot consist of only blank characters',
   'er.input.invalid' => 'Invalid character found',
   'er.input.range' => 'Value not in permitted range',
+  'er.input.range.len' => 'Value not in permitted range, max length permitted "%1$s"',
```

Comment:
I would probably just call this `er.input.len`, since `range` implies something like "value must be between 1 and 10".

Reply:
Done